internal/controller/orchestrator_controller.go

```diff
@@ -137,144 +137,126 @@ func (r *OrchestratorReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		}
 	}
 
+	argoCDEnabled := orchestrator.Spec.ArgoCd.Enabled
+	tektonEnabled := orchestrator.Spec.Tekton.Enabled
+	wfNamespace := orchestrator.Spec.OrchestratorConfig.Namespace
+
 	// handle sonataflow
-	sonataFlowOperator := orchestrator.Spec.SonataFlowOperator
+	sonataFlowOperator := orchestrator.Spec.ServerlessLogicOperator
 	if err = r.reconcileSonataFlow(ctx, sonataFlowOperator, orchestrator); err != nil {
 		logger.Error(err, "Error occurred when installing SonataFlow resources")
 		_ = r.UpdateStatus(ctx, orchestrator, orchestratorv1alpha1.FailedPhase, metav1.Condition{
-			Type:    TypeDegrading,
-			Status:  metav1.ConditionFalse,
-			Reason:  "Failed to create SonataFlow Resources",
-			Message: err.Error(),
+			Type:               TypeDegrading,
+			Status:             metav1.ConditionFalse,
+			Reason:             "ReconcilingSonatflowResourcesFailed",
+			Message:            err.Error(),
+			LastTransitionTime: metav1.Now(),
 		})
-		return ctrl.Result{RequeueAfter: 1 * time.Minute}, err
+		return ctrl.Result{RequeueAfter: 5 * time.Minute}, err
 	}
-	_ = r.UpdateStatus(ctx, orchestrator, orchestratorv1alpha1.CompletedPhase, metav1.Condition{
-		Type:    TypeProgressing,
-		Status:  metav1.ConditionTrue,
-		Reason:  "Created SonataFlow Resources",
-		Message: "Completed SonataFlow Reconciliation",
-	})
+	//_ = r.UpdateStatus(ctx, orchestrator, orchestratorv1alpha1.CompletedPhase, metav1.Condition{
+	//	Type:               TypeProgressing,
+	//	Status:             metav1.ConditionTrue,
+	//	Reason:             "ReconcilingSonatflowResourcesCompleted",
+	//	Message:            "Completed SonataFlow Reconciliation",
+	//	LastTransitionTime: metav1.Now(),
+	//})
 
 	//handle knative
 	serverlessOperator := orchestrator.Spec.ServerlessOperator
 	if err := r.reconcileKnative(ctx, serverlessOperator); err != nil {
 		logger.Error(err, "Error occurred when installing K-Native resources")
 		_ = r.UpdateStatus(ctx, orchestrator, orchestratorv1alpha1.FailedPhase, metav1.Condition{
-			Type:    TypeDegrading,
-			Status:  metav1.ConditionFalse,
-			Reason:  "Failed to create K-Native Resources",
-			Message: err.Error(),
+			Type:               TypeDegrading,
+			Status:             metav1.ConditionFalse,
+			Reason:             "ReconcilingKNativeResourcesFailed",
+			Message:            err.Error(),
+			LastTransitionTime: metav1.Now(),
 		})
-		return ctrl.Result{Requeue: true, RequeueAfter: 3 * time.Minute}, err
+		return ctrl.Result{RequeueAfter: 5 * time.Minute}, err
```

Comment:
`5` should be a constant

Reply:
Updated to constant in [commit](https://github.com/parodos-dev/orchestrator-go-operator/pull/3/commits/8c288dd87483b02e257fd3670666d25cae7ad97b)